schemas/iso19139/src/main/plugin/iso19139/index-fields/default.xsl

```diff
@@ -896,9 +900,18 @@
 
     <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
-    <xsl:for-each select="gmd:language/gco:CharacterString
```

Comment:
For metadata language cardinality is [0..1], probably in most cases will be defined, but without the loop can be indexing empty data I guess, in case no language defined.